### PR TITLE
navigation bar

### PR DESCRIPTION
**Pull request by tabassum2507**
@rohan-arya-w Review the Navigation Bar Code

### CODE REVIEW

**Review by emailpocketxtra — COMMENTED** [review]
Split Interfaces in seperate  files --- src/interface/navigationBar/navigationBarInterface.ts

**Review by rohan-arya-w — APPROVED** [review]

File: src/interface/navigationBar/navigationBarInterface.ts
Comment (Amitrawat21) [inline src/interface/navigationBar/navigationBarInterface.ts:1]: write in a  sequence

Comment (Amitrawat21) [reply src/interface/navigationBar/navigationBarInterface.ts:1]: write mandatory in icon

File: src/interface/navigationBar/navigationBarInterface.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export interface navigationBarInterface {
```
Comment (ShubhamBade) [inline src/interface/navigationBar/navigationBarInterface.ts]: make interface first letter in caps

Comment (tabassum2507) [reply src/interface/navigationBar/navigationBarInterface.ts]: Resolved

**Review by ShubhamBade — APPROVED** [review]
done.

**Pull request merged**